content/doc/developer/testing/index.adoc

```diff
@@ -2,15 +2,331 @@
 title: Testing
 layout: developerchapter
 wip: true
+# TODO: Why aren't these appearing?
```

Comment:
When I ran `make run` the references appear.

One oddity you might be hitting with the auto-reload in awestruct, if you change the YAML Frontmatter for a document after awestruct has already processed the file, it won't automatically re-initialize the datastructure, just re-render the content of the file (after the frontmatter).

I would presume that if you stopped `make run` and restarted it, you would see the References.

Reply:
They're fixed with the above commit — I forgot to remove this comment… 🤦‍♂️